backend/app/Controllers/User/TicketsController.php

```
@@ -0,0 +1,1073 @@
+        if ($fileSize > 50 * 1024 * 1024) {
+            return ApiResponse::error('File size too large. Maximum size is 50MB', 'FILE_TOO_LARGE', 400);
+        }
+
+        // Create attachments directory if it doesn't exist
+        $attachmentsDir = APP_PUBLIC . '/attachments/';
+        if (!is_dir($attachmentsDir)) {
+            mkdir($attachmentsDir, 0755, true);
+        }
+
+        // Generate unique filename
+        $extension = $file->guessExtension() ?: pathinfo($originalName, PATHINFO_EXTENSION);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$extension'. (undefined)

(UnusedLocalVariable)